FizzBuzz.py

```diff
@@ -0,0 +1,49 @@
+#FizzBuzz by M.Horsler 4/10/23. Utilise Lists to allow manipulation and concatenation to provide desired result.
+#Print to Screen function
+def Screen(text):
+    print (text)
+
+#Define variables
+Loop = 1
+result_list=[]
+
+while Loop<256:
+ #Ensure Bong overrides all by processing first:
+    if Loop % 11 == 0:
+        result_list.append('Bong')
+    else:
+        if Loop % 3 == 0:
+            result_list.append('Fizz')
+
+        if Loop % 13 == 0:
+            result_list.append('Fezz')
+
+# Buzz processed after Fizz to create desired order.
+        if Loop %3==0 and Loop%5==0:
+            result_list.append('Buzz')
+
+        elif Loop % 5 == 0:
+            result_list.append('Buzz')
```

Comment:
I think we could combine these since the action is identical, literally:
```python
        if (Loop %3==0 and Loop%5==0) or Loop % 5 == 0:
            result_list.append('Buzz')
```
But if we look carefully that first case can only be true if the second one is true*, so that can be simplified further to just:
```suggestion
        if Loop % 5 == 0:
            result_list.append('Buzz')
```

*If you're not sure about this, the first case can be rewritten as `if (Loop%5==0) and (Loop%3 == 0 or Loop%3 != 0)` where that second part is obviously True always!